packages/protocol/contracts/bridge/IBridge.sol

```diff
@@ -80,3 +80,9 @@ interface IRecallableSender {
         external
         payable;
 }
+
+/// @title IMessageInvocable
+/// @notice An interface that all bridge message receiver shall implement
+interface IMessageInvocable {
+    function onMessageInvocation(bytes calldata data) external payable;
+}
```

Comment:
The `CrossChainOwned` contract still needs to be updated to work with this new interface.

Reply:
Good catch!